packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-fundamentals.md

```diff
@@ -0,0 +1,100 @@
+---
+id: About
+section: accessibility
+---
+
+# Accessibility fundamentals 
+
+Accessibility is a foundation of PatternFly, so if you contribute to PatternFly or use it for your products you should first familiarize yourself with our accessibility guidelines and recommendations.
+
+No user should feel left behind when using your product. The goal of accessible design is to remove barriers and create inclusive product experiences that work for everyone, regardless of ability. This is best achieved when accessibility is considered early in the design and development process. 
+
+There’s always room to improve accessibility. That’s why our guidelines will continually evolve. Your feedback can help support PatternFly's accessibility, so we welcome you to contribute to our documentation or [get involved in our GitHub discussions](https://github.com/orgs/patternfly/discussions). 
+
+## Assistive technologies
+
+A core consideration for accessibility practices is assistive technologies that people may use to access your product. 
+
+### Keyboard
+Some users can’t use a mouse and rely on other methods of navigation instead, like a keyboard. A keyboard allows users to navigate through UI elements in the order that they appear on the page, typically using the tab key. Keyboard users should be able to do anything that mouse users can, including viewing text that is shown upon hover or in popups.  
+
+### Screen readers
+When a user has impaired vision (temporary or permanent) that prevents them from viewing a screen, they can use a screen reader that reads visible and non-visible content aloud. Non-visible content includes hidden text, like tooltips, and alternative text (alt text) for icons, images, and links. 
+
+## Understanding users’ needs
+Great user experiences don’t just happen. They’re designed, tested, and refined with users in mind. To develop inclusive products, it’s important to understand the varying needs of all users and consider the assistive tools and methods that they use.
+
+This section provides information to help you better understand and address the needs of users [with diverse abilities](https://www.a11yproject.com/posts/accessibility-is-blind-people/). It is important to note that some users might fall into multiple groups, and some might use tools created for a different group.
+
+### Motor control
+Users with reduced motor control often use a keyboard or computer mouse to access content.
```

Comment:
Not 100% about the wording, but what do you think about an addition similar to the following? It kind of ties into the bullet about target area size.


```suggestion
Users with reduced motor control often use a keyboard or computer mouse to access content. Some users may also have difficulty keeping a mouse steady when trying to interact with content.
```